aries-site/src/data/structures/templates/templates.js

```diff
@@ -244,6 +245,26 @@ export const templates = [
     sections: [],
     tags: [],
   },
+  {
+    name: 'Popover',
+    available: true,
+    cardOrder: 5,
+    description: `A Popover is an overlay that is opened by a
+     click trigger.
+     They are used to provide additional contextual information and might
```

Comment:
```suggestion
     It is used to provide additional contextual information and might
```

Do we need to include the "and might contain interactive elements" in this initial description? Feels a bit excessive but soft opinion.